civis/service_client.py

```diff
@@ -28,6 +28,40 @@ def auth_service_session(session, client):
     session.get(auth_url)
 
 
+def _parse_service_path(path, operations, service_client=None):
```

Comment:
How about having this function just take "root_path" or something like?  Passing in the whole service_client object seems unnecessary.